docs/modules/core/pages/vm-migrate-volume-specs.md

```
@@ -0,0 +1,250 @@
+| 5 | Shareable 卷拒绝（沿用 ShareableVolumeVmInstanceRefVO） |
+| 6 | specs **不含 Root** 时（仅数据盘场景）：每个 dstPS 必须挂在 VM 当前 host 的 cluster；通过 `checkVmNicClusterCompatibility` |
+| 7 | specs **含 Root** 且 VM Running/Paused：按 metric `isSupportLive` 校验热迁能力 |
+| 8 | specs **含 Root** 且 VM Stopped：按 metric `isSupportOffline` 校验 |
+| 9 | 每条 spec 单独校验 src/dst PS Enabled、`metric.isCapable(src.type, dst.type)`、L2 兼容性 |
+| 10 | `dstHostUuid` 非空时：必须属于 Root spec 的 dstPS 关联 cluster（若有 Root spec） |
+
+---
+
+## 5. 执行链路
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)